src/binder_util.h

```diff
@@ -117,7 +117,8 @@ binder_access_modes_from_raf(
 
 enum ofono_radio_access_mode
 binder_access_modes_up_to(
-    enum ofono_radio_access_mode mode)
+    enum ofono_radio_access_mode mode,
+    enum ofono_radio_access_mode supported_modes)
```

Comment:
Same here. With `enum` you define one value only. If you need multiple values packed into one it might be clearer to use other type implicating bit shifting is required.